src/render.js

```diff
@@ -138,15 +171,17 @@ const initSettings = () => {
 
   // Populate the dialect selector
   const dialectSelector = document.getElementById('dialectSelection');
-  Generator.SUPPORTED_DIALECTS.forEach((language) => {
-    const opt = document.createElement('option');
-    opt.value = language;
-    opt.innerHTML = language;
-    dialectSelector.appendChild(opt);
-    if (gherkinDialect === language) {
-      opt.selected = 'selected';
-    }
-  });
+  if (Generator.SUPPORTED_DIALECTS) {
```

Comment:
the only reason these wouldn't exist is if the required report-generator package doesn't have it, which would only occur if the required version was changed to a lower version before the field existed?